File: scripts/archive/root-noise-2026-04-13/tmp-find2.mjs
import fs from 'fs';

const content = fs.readFileSync(
  'D:/project/xinjuben/src/main/application/script-generation/prompt/create-script-generation-prompt.ts',
  'utf8'
);

const keywords = [
  '耳边回响',
  '脑海中浮现',
  '不可拍心理句',
  '分析人物',
  '少年守钥人',
  '旧三段',
];

for (const kw of keywords) {
  const idx = content.indexOf(kw);
  if (idx >= 0) {
    console.log(`FOUND "${kw}" at ${idx}:`);
    console.log(`  ...${content.substring(Math.max(0, idx-20), idx+kw.length+20)}...`);
    console.log();
  } else {
    console.log(`MISSING: "${kw}"`);
  }
}

File: scripts/archive/root-noise-2026-04-13/tmp-find2.py
with open('D:/project/xinjuben/src/main/application/script-generation/prompt/create-script-generation-prompt.ts', 'r', encoding='utf-8-sig') as f:
    content = f.read()
idx = content.find("if (storyContract.requirements.requireRelationshipShift)")
if idx >= 0:
    print(f"Found at {idx}:", repr(content[idx:idx+300]))
else:
    print("Not found. Searching alternatives...")
    for keyword in ['requireRelationshipShift', 'relationshipShift', 'requireRelationship']:
        idx2 = content.find(keyword)
        if idx2 >= 0:
            print(f"Found '{keyword}' at {idx2}:", repr(content[idx2:idx2+100]))
            break
    else:
        print("None found")
